seonoh/ArchitectureStudy/app/src/main/java/com/example/seonoh/seonohapp/TabPagerAdapter.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.seonoh.seonohapp
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentStatePagerAdapter
+import com.example.seonoh.seonohapp.MainActivity.Companion.btcMarketData
+import com.example.seonoh.seonohapp.MainActivity.Companion.ethMarketData
+import com.example.seonoh.seonohapp.MainActivity.Companion.krwMarketData
+import com.example.seonoh.seonohapp.MainActivity.Companion.usdtMarketData
+
+class TabPagerAdapter(fm: FragmentManager, tabcount: Int) : FragmentStatePagerAdapter(fm) {
+
+    private val tabCount = tabcount
+
+
+
+    override fun getItem(position: Int): Fragment {
+        when (position) {
```

Comment:
```suggestion
return when (position) {
            0 -> CoinFragment.newInstance(KRW_TYPE, krwMarketData)

            1 -> CoinFragment.newInstance(BTC_TYPE, btcMarketData)

            2 -> CoinFragment.newInstance(ETH_TYPE, ethMarketData)

            3 -> CoinFragment.newInstance(USDT_TYPE, usdtMarketData)

            else -> throw Throwable()
        }
```
위와 같이 변경 가능할 것 같습니다.
마켓 정보(KRW, BTC, ETH, USDT)보두 viewType이 같기 때문에 굳이 viewType을 각각 나눌 필요가 없을 것 같습니다.

Reply:
감사합니다 종선님 !